build.sh

```diff
@@ -1,7 +1,19 @@
-git clone https:\/\/github.com\/fossasia\/fossasia.org.git out
+git config --global push.default simple
+ssh-agent bash -c "ssh-add ${travis_deploy_key}"
```

Comment:
The issues I am having now are: 

1. authentication is not working (possibly because the secure env vars aren't exposed to fork builds)
2. Because I am building from a fork, there isn't any environment variable for the repo slug for the fork being built. (TRAVIS_REPO_SLUG seems to address the main repository.

Please advise. @niccokunzmann @mariobehling